src/MessageOutbox/RelayMessagesThroughConsumerTest.php

```diff
@@ -11,7 +11,7 @@
 
 use function iterator_to_array;
 
-class OutboxRelayTest extends TestCase
+class RelayMessagesThroughConsumerTest extends TestCase
```

Comment:
I can copy the test and keep `OutboxRelayTest` there if you prefer (idk if you want to make sure `OutboxRelay` is still tested).

Reply:
No need, it'd bring hardly any value.